src/main/java/core/basesyntax/dao/ReaderDataFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public interface ReaderDataFromFile {
```

Comment:
```suggestion
public interface FileDataReader{
```
we can usually reorder words to avoid using From